src/renderer.rs

```diff
@@ -165,6 +167,12 @@ pub fn page(
                             }
                         }
                     }
+            @if readme.is_some() {
+                div {
+                    h3 { (readme.as_ref().unwrap().filename) }
```

Comment:
Do you still need this construct now that you have the special struct for it?

Reply:
Since `readme` here is `Option<Readme>` looks like it'll be moved when I use `.unwrap()` so to access two fields separately I need to use `.as_ref()` the first time. Other option would be to save unwrapped value to a variable I guess.

Reply:
I think we can probably find a simpler solution here. It doesn't prevent me from merging but I might golf it some more.

Reply:
Probably, but considering previous code I think we've made it a lot straightforward, so, I'm satisfied.